### ISSUE TITLE
Scrolling/scrollwheel disabled when using pen tablet (Huion Kamvas 16)

### ISSUE THREAD

**Issue by cbertram-zz** [glimpse-editor/Glimpse#396]
### Describe the bug
When I draw using the mouse I am able to scroll around the canvas or zoom in/out using the Huion Kamvas 16 push buttons or scroll bar. When I move the cursor by moving the pen around on the tablet the tablet, both the push buttons/scroll bar and the scroll wheel on the mouse cease to work to move or zoom the canvas.

### To Reproduce
1. Set push buttons on Huion pen tablet to "mouse wheel scroll up" and/or "mouse wheel scroll down".
2. Open Glimpse and enable external input devices.
2. Open a new canvas and zoom in.
3. Move the cursor using the pen on the Huion tablet.
4. Attempt to scroll.

### Expected behavior
Canvas should scroll up/down with mouse wheel moving forwards/backwards or external input of the same.

### Desktop (please complete the following information)

Windows 10, Glimpse 0.1.2 and Huion drivers 14.8.90

### Additional context
Scroll function works elsewhere within the program - e.g. scaling brush property values, scrolling through menus.

Same issue exists in GIMP but not Adobe Illustrator CS2.

Sometimes moving the mouse seems enough to reinstate scrolling, sometimes drawing on the canvas, sometimes I have to switch tool with the mouse before drawing on the canvas works.

**Comment by Member1221** [comment]
Nobody in the development team currently have a drawing tablet, so it won't be something we can reproduce on our end unless someone with a drawing tablet and coding knowledge steps in. Or one of us acquires one at some point.

For that reason I can't specify a point where we can fix this bug, sorry.

**Comment by TrechNex** [comment]
Indeed, this isn't the first time Windows-based tablet users have had problems, eg. #229 #389

That is a problem we have inherited from upstream, and unfortunately we lack the capacity to fix it ourselves. My apologies.

(As an aside, I do technically have a graphics tablet, but it's a very old Wacom I bought specifically because it's well-supported!)

**Comment by cbertram-zz** [comment]
Thanks for the rapid replies! This is my first bug report on Git :) I appreciate the problems in testing this, I just wanted to get it logged.

I've just tried to program the button to "left mouse click" and that doesn't seem to work when I start to move the pen either. Maybe using the external device is disabling the Core Pointer, including the buttons?

**Comment by Member1221** [comment]
I think it has to do with how input works in upstream, the way it recognizes between stuff like tablet pen, mouse and tablet pen eraser is through multiple inputs called different things. It might be related to that?

Not something I can really experiment with, though ^^;
